src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -1141,6 +1131,7 @@ InstanceKlass* ClassLoader::load_class(Symbol* name, PackageEntry* pkg_entry, bo
     } else {
       // Exploded build - attempt to locate class in its defining module's location.
       assert(_exploded_entries != nullptr, "No exploded build entries present");
+      assert(!CDSConfig::is_dumping_archive(), "CDS doesn't support exploded build");
```

Comment:
Suggestion: update the assert comment to `"CDS dumping doesn't support exploded build"`.

Reply:
Fixed. Thanks for the review.